src/main/java/dev/aurelium/auramobs/util/ScaleUtil.java

```diff
@@ -0,0 +1,114 @@
+package dev.aurelium.auramobs.util;
+
+import dev.aurelium.auramobs.AuraMobs;
+import org.bukkit.attribute.Attribute;
+import org.bukkit.configuration.ConfigurationSection;
+import org.bukkit.entity.LivingEntity;
+
+import java.util.Arrays;
+import java.util.LinkedList;
+import java.util.concurrent.ThreadLocalRandom;
+
+public class ScaleUtil {
+
+    private static final LinkedList<ScaleEntry> entries = new LinkedList<>();
```

Comment:
Use a `List<ScaleEntry> entries = new ArrayList<>()` instead, LinkedList is less performant.